tests/modules/programs/aerc/hm-example.com-msmtp-expected.conf

```diff
@@ -0,0 +1,33 @@
+# Generated by Home Manager.
+set ssl_force_tls = yes
+set certificate_file=/etc/ssl/certs/ca-certificates.crt
+
+# GPG section
+set crypt_use_gpgme = yes
+set crypt_autosign = no
+set pgp_use_gpg_agent = yes
+set mbox_type = Maildir
+set sort = "threads"
+
+# MTA section
+set sendmail='msmtpq --read-envelope-from --read-recipients'
```

Comment:
The overlay for msmtp will need to set `outPath = "@msmtp@";` and then this will have to be updated:

``` suggestion
set sendmail='@msmtp@/bin/msmtpq --read-envelope-from --read-recipients'
```